Makefile

```diff
@@ -5,7 +5,9 @@
 .PHONY: clean
 
 build:
-	 ./node_modules/.bin/node-gyp configure && ./node_modules/.bin/node-gyp build
+	curl -s https://publicsuffix.org/list/public_suffix_list.dat -o etld/data/public_suffix_list.dat
```

Comment:
Kudos for not using `-k` here :)

nit: If you want to make that `curl` line even stricter, you could also throw in `--tlsv1.2` to enforce a minimum level of TLS.